common/src/main/java/com/microsoft/identity/common/internal/controllers/LocalMSALController.java

```diff
@@ -437,7 +437,7 @@ private void renewAT(@NonNull final SilentTokenCommandParameters parametersWithS
                          @NonNull final ICacheRecord cacheRecord,
                          @NonNull final String tag) throws IOException, ClientException, ServiceException {
         Logger.verbose(
-                TAG + tag,
+                tag,
```

Comment:
what are the values for this?, should this be part of the tag or message

Reply:
This method is invoked in multiple locations in the class and is more of a helper, so we need to get the tag from the calling method.